--- ../../trelis_sdk/Coreform-Cubit-2020.2/Coreform-Cubit-2020.2.app/Contents/MacOS/CubitGeomConfig.cmake	2020-11-10 12:52:31.000000000 -0600
+++ CubitGeomConfig.cmake	2021-05-14 15:42:06.858956001 -0500
@@ -4,7 +4,7 @@
    message(FATAL_ERROR "CMake >= 2.6.0 required")
 endif()
 cmake_policy(PUSH)
-cmake_policy(VERSION 2.6...3.17)
+cmake_policy(VERSION 2.6)
 #----------------------------------------------------------------
 # Generated CMake target import file.
 #----------------------------------------------------------------
@@ -54,7 +54,7 @@
 add_library(cubit_geom SHARED IMPORTED)
 
 set_target_properties(cubit_geom PROPERTIES
-  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include/geom"
+  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/Trelis-17.1.app/Contents/include/geom"
   INTERFACE_LINK_LIBRARIES "cubit_util"
   INTERFACE_PRECOMPILE_HEADERS "\$<\$<COMPILE_LANGUAGE:CXX>:RefEntity.hpp>;\$<\$<COMPILE_LANGUAGE:CXX>:Body.hpp>;\$<\$<COMPILE_LANGUAGE:CXX>:RefVolume.hpp>;\$<\$<COMPILE_LANGUAGE:CXX>:RefFace.hpp>;\$<\$<COMPILE_LANGUAGE:CXX>:RefEdge.hpp>;\$<\$<COMPILE_LANGUAGE:CXX>:RefVertex.hpp>;\$<\$<COMPILE_LANGUAGE:CXX>:GeometryQueryTool.hpp>;\$<\$<COMPILE_LANGUAGE:CXX>:GeometryModifyTool.hpp>"
 )
@@ -92,7 +92,7 @@
 endforeach()
 unset(_IMPORT_CHECK_TARGETS)
 
-# Make sure the targets which have been exported in some other
+# Make sure the targets which have been exported in some other 
 # export set exist.
 unset(${CMAKE_FIND_PACKAGE_NAME}_NOT_FOUND_MESSAGE_targets)
 foreach(_target "cubit_util" )
